### ISSUE TITLE
CDN link for browser.

### ISSUE THREAD

**Issue by webdev3301** [chrisguttandin/worker-timers#423]
Hello. Is there a CDN link for devs who don't use node js? What if someone wants to create a browser app in the browser. No server side. does this still work?

**Comment by webdev3301** [comment]
Ahhh... webpack. Hard pass.

**Comment by chrisguttandin** [comment]
Yeah, I know it's not everyone's favorite tool. Does using JSPM as shown in the example above work for you?

**Comment by ROBERT-MCDOWELL** [comment]
I'm looking for also a simple JS minified link to import as <script src....></script>
pure JS is the best for me.

**Comment by babaric-dev** [comment]
Hello. Any updates for this request? I am looking for this too.

**Comment by YellowMaker** [comment]
Any hope for simple `<script src="worker-timer.js"></script>` version?

**Comment by PLTytus** [comment]
I made something. Hope, this will help some folks.
https://github.com/PLTytus/worker-timers-cdn
